modules/aerodyn/src/AeroDyn_Inflow.f90

```diff
@@ -299,9 +299,10 @@ subroutine ADI_CalcOutput(t, u, p, x, xd, z, OtherState, y, m, errStat, errMsg)
 
    ! --- Set outputs
    !TODO: this assumes one rotor!!!
-   associate(AD_NumOuts => p%AD%rotors(1)%NumOuts + p%AD%rotors(1)%BldNd_TotNumOuts)
-      y%WriteOutput(1:AD_NumOuts) = y%AD%rotors(1)%WriteOutput(1:AD_NumOuts)
-      y%WriteOutput(AD_NumOuts+1:p%NumOuts) = y%IW_WriteOutput(1:m%IW%p%NumOuts)
+   associate(AD_NumOuts => p%AD%rotors(1)%NumOuts + p%AD%rotors(1)%BldNd_TotNumOuts, &
+             IW_NumOuts => m%IW%p%NumOuts)
+      y%WriteOutput(1:IW_NumOuts) = y%IW_WriteOutput(1:m%IW%p%NumOuts)
```

Comment:
You could change the second `m%IW%p%NumOuts` to `IW_NumOuts` as well.

Reply:
That's a good point, will do.